string cmd
float64[] float_ary
geometry_msgs/Twist pose
string req_s
---
string result
string res_s
float64 f
geometry_msgs/Twist pose

